webservices/common/models/dates.py

```diff
@@ -1,18 +1,39 @@
+import re
+
+from sqlalchemy import ForeignKey
+
 from webservices import decoders
 
 from .base import db
 
+class ReportNames(db.Model):
+    __tablename__ = 'dimreporttype'
```

Comment:
Didn't notice this earlier, but it would be best to avoid using tables from the data warehouse in new functionality. Let's ask FEC if there's an equivalent table in FECP that we can use instead.